Dockerfile

```diff
@@ -2,6 +2,11 @@ FROM registry.centos.org/kbsingh/openshift-nginx:latest
 
 MAINTAINER Jean-Francois Chevrette <jfchevrette@gmail.com>
 
+# Labels consumed by the build service
+LABEL Component="nginx-redirector" \
+      Name="mattermost/nginx-redirector" \
```

Comment:
Would like to keep this image generic. Is it possible to remove 'mattermost/' and just have nginx-redirector as a name?